nucypher/characters/lawful.py

```diff
@@ -103,6 +102,16 @@ def __init__(self,
 
         self.active_policies = dict()
 
+    def add_active_policy(self, active_policy):
+        """
+        Adds a Policy object that is active on the NuCypher network to Alice's
```

Comment:
This is excellent for auto-generated docs. :+1:

Reply:
Um... I wrote that lol

Reply:
Yes, or course - I just pointing out that the addition of this method is good for the docs.